interfaces/mount/entry.go

```diff
@@ -103,3 +119,28 @@ func (e Entry) String() string {
 	return fmt.Sprintf("%s %s %s %s %d %d",
 		name, dir, fsType, options, e.DumpFrequency, e.CheckPassNumber)
 }
+
+// ParseEntry parses a fstab-like entry.
+func ParseEntry(s string) (Entry, error) {
+	var e Entry
+	fields := strings.Fields(s)
+	// do all error checks before any assignments to `e'
+	if len(fields) != 6 {
```

Comment:
The last two fields (fs_freq and fs_passno) are optional and default to `0` if missing. So this needs to be adjusted (happens in the real world).

Also it appears that sometimes fstab entries with more fields exist in the wild, e.g. https://bugs.launchpad.net/ubuntu/+source/update-manager/+bug/873411/comments/7 - we may want to ignore those, this is similar to what glibc is doing).